rotkehlchen/db/schema.py

```diff
@@ -437,6 +437,16 @@
 );
 """
 
+DB_CREATE_OPTIMISM_TRANSACTIONS = """
```

Comment:
Can you please also add a step in the DB upgrade where new tables are created and add this exact same table creation there?

Or just add a TODO so that I or me do it later.

WE will also need a data migration to update all existing optimism transactions to conform to the new schema. As it can't happen during the upgrade since it's too many network calls.

But that should really be a separate PR.

Reply:
I wasn't 100% sure what I needed to do here, I just added the TODO

Reply:
Oh it's simple. So these commands in db/schema.py only run for the new creation of the DB. Which would happen for a new user. For all existing users, when there is any sort of modification of the schema, and a new DB table is one such modification, it needs to be added to the DB upgrade.

The latest DB upgrade is this: https://github.com/rotki/rotki/blob/develop/rotkehlchen/db/upgrades/v38_v39.py . It will be included in our next release from the develop branch (1.30.0).
And you should just add a new step with a new function to create new tables and insert this one you made there.

But just the TODO is fine also since I am touching all this code in my PR anyway and would need to change it again so it's fine.

But just letting you know so you know what I meant. One such create_new_tables step is here for the v36->v37 DB upgrade: https://github.com/rotki/rotki/blob/63d49f2501114376d5a9f727b083b280fa092f4e/rotkehlchen/db/upgrades/v36_v37.py#L166